Sources/Auth/Services/Signer/MessageSigner.swift

```diff
@@ -1,11 +1,11 @@
 import Foundation
 
 protocol MessageSignatureVerifying {
-    func verify(signature: String, message: String, address: String) throws
+    func verify(signature: CacaoSignature, message: String, address: String) throws
 }
 
 protocol MessageSigning {
-    func sign(message: String, privateKey: Data) throws -> String
+    func sign(message: String, privateKey: Data) throws -> CacaoSignature
 }
 
 public struct MessageSigner: MessageSignatureVerifying, MessageSigning {
```

Comment:
MessageSigner sounds very generic but the signer is binded to CACAO and EIP4361. We can consider renaming the class and protocol.